lib/active_admin/orm/active_record/comments/views/active_admin_comments.rb

```diff
@@ -46,6 +49,14 @@ def build_empty_message
           span I18n.t('active_admin.comments.no_comments_yet'), class: 'empty'
         end
 
+        def comments_url(*args)
+          parts = []
+          parts << active_admin_namespace.name unless active_admin_namespace.root?
+          parts << active_admin_namespace.comments_registration_name.underscore
+          parts << 'path'
+          send parts.join('_'), *args
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.